backend/app/users/routers.py

```diff
@@ -42,8 +43,20 @@ async def create_user(
     """
     Create user endpoint.
     """
-
     try:
+        # Import workspace functionality to avoid circular imports
+        from ..workspaces.models import (
```

Comment:
We also have models and schemas a level up. We should discuss how to refactor this